### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by Bondok6**
- Add Js linters (.eslintrc.json) file.
- Add main.js file
- Add mobile menu functionality

      - When the user clicks (or taps) the hamburger button, the mobile menu appears.
      - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
      - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
      - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

- Add a smooth transition.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "import/extensions": [
+      1,
+      {
+        "js": "always",
+        "json": "always"
+      }
+    ]
+  },
+  "ignorePatterns": ["dist/", "build/"]
+}
```
Comment (rotimiazeez) [inline .eslintrc.json:26]: - [x] Kindly update your github/workflows to use the html-css-js linter.yml config, so you can check javascript linter too😎 you can find it [here ](https://github.com/microverseinc/linters-config)✌

File: styles/main.css [hunk 2]
```diff
@@ -145,7 +145,28 @@ nav {
...
+  top: 8vh;
+  left: 0;
+  width: 100vw;
+  height: 92vh;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  transition: transform 0.5s ease;
+  transform: translateX(-100%);
```
Comment (rotimiazeez) [inline styles/main.css]: - [x] Kindly make sure your design matches the original Figna design✌
- Your Design
screenshot (7)
- Figma Design
screenshot (8)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👋, 
## Good job so far!🔥
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofalconi — APPROVED** [review]
Hi @team,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**